pygame/mask.py

```diff
@@ -1,6 +1,6 @@
 # Implement the pygame API for the bitmask functions
 
-from math import atan2, pi
+import math
```

Comment:
mask.py is intending to import the standard math module - this code was there before the local math module was added.  Should I still add the absolute_import?  That's default behaviour isn't it, or are we aiming to support Python < 2.7?